cli-staking/src/args.rs

```diff
@@ -32,12 +32,7 @@ pub struct ConfigOverride {
 }
 
 #[derive(Parser, Debug)]
-pub struct VaultArgs {
-    #[clap(long)]
-    /// Base keypair file required to initialize the vault
-    ///
-    /// /path/to/base/keypair.json
-    pub base: String,
+pub struct VaultArgs {    
```

Comment:
We can update `Init` command to use only `token_mint`, and drop `VaultArgs` as `base` will be generated on the fly. Something like:
```
    Init {
        #[clap(long)]
        token_mint: Pubkey
    }
```

Reply:
Thanks, it has be fixed